src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html class="html" lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg" 
+           alt="product photo" 
+           class="card__image">
+    
+      <h2 class="card__product_name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <h4 class="card__product_id">Код товара: 195434</h4>
+      <div class="card__rate">
+        <div class="card__rate_stars">
+          <div class="stars-position card__rate_stars-active"></div>
```

Comment:
use BEM, don't use active class for stars, use modifier with number of active stars
reuse styles from the previous task (stars)

```suggestion
        <div class="card__stars stars stars--4">
          <div class="stars__star"></div>
```